joomla/joomla-cms
# [4.0] Namespace com_cache

Pull Request for Issue # .

### Summary of Changes
This simple PR convert com_cache component to namespace MVC for Joomla 4.

### Testing Instructions
Overall, test to make sure it works in the same as before (none namespace version)

1. Install latest Joomla 4.0-dev with sample data (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_cache.zip )

2. Go to System -> Global Configuration, look at System tab, enable Cache

3. Navigate to some pages in frontend of the site so that cache can be created.

4. Access to System -> Clear Cache, try to delete cache, make sure it is working as expected
